src/gsMpi/gsMpiComm.h

```diff
@@ -141,6 +144,68 @@ class gsSerialComm
         return 0;
     }
 
+    /** @brief Sends data to a destination process with a defined tag (blocking)
+     *
+     * One process sends data of length len to the destination process dest. The argument tag specifies the message ID.
+     *
+     * @param[in] in The send buffer with the data to send
+     * @param[in] len The number of elements which will be sent
+     * @param[in] dest The rank of the process which should receive the message
+     * @param[in] tag Specifies the message ID
+     */
+    template<typename T>
+    static int send (T* in, int len, int dest, int tag = 0)
+    {
+        return 0;
+    }
+
+    /** @brief Sends data to a destination process with a defined tag (non-blocking)
+     *
+     * One process sends data of length len to the destination process dest. The argument tag specifies the message ID.
+     *
+     * @param[in] in The send buffer with the data to send
+     * @param[in] len The number of elements which will be sent
+     * @param[in] dest The rank of the process which should receive the message
+     * @param[out] request communication request
+     * @param[in] tag Specifies the message ID
+     */
+    template<typename T>
+    static int isend (T* in, int len, int dest, MPI_Request request, int tag = 0)
+    {
+        return 0;
+    }
+
+    /** @brief Receives data from a source process with a defined tag (blocking)
+     *
+     * One process receives data of length len from the source process source. The argument tag specifies the message ID.
+     *
+     * @param[out] out The buffer to store the received data in
+     * @param[in] len The number of elements which will be received
+     * @param[in] source The rank of the process which sended the message
+     * @param[in] tag Specifies the message ID
+     */
+    template<typename T>
+    static int recv (T* out, int len, int source, MPI_Status status, int tag = 0)
+    {
+        return 0;
+    }
```

Comment:
This one is has a problem:
https://cdash.ricam.oeaw.ac.at/viewBuildError.php?onlydeltap&buildid=245103